### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
Added example of usage, installation and testing sections to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,62 @@ pymets [![Build Status](https://travis-ci.org/unt-libraries/pymets.svg?branch=ma
...
+        <name>UNT Libraries: Digital Projects Unit</name>
+    </agent>
+  </metsHdr>
+</mets>"""
+>>> res = metsdoc.metsxml2py(io.BytesIO(mets_string.encode('utf-8')))
+```
+
+Requirements
+-------------
+* python ~=3.7
```
Comment (ldko) [inline README.md]: The tests also pass on 3.6. We could change this to below and add `py36` environment to tox testing.
```suggestion
* Python 3.6-3.7
```

File: README.md [hunk 2]
```diff
@@ -3,6 +3,62 @@ pymets [![Build Status](https://travis-ci.org/unt-libraries/pymets.svg?branch=ma
...
+>>> # To create a METS object
+>>> attributes = {
+            'TYPE': 'archival information package',
+            'OBJID': 'ark:/67531/12345',
+        }
+>>> m = mets_structure.Mets()
+>>> m.set_atts(attributes)
+>>> m.add_child(mets_structure.MetsHdr())
+>>> print(m.create_xml_string())
+>>> <?xml version="1.0" encoding="UTF-8"?>
```
Comment (ldko) [inline README.md]: This is the output, so it shouldn't have the `>>>`. I suggest showing the output like it looks in the interpreter with `b''` around it etc.:
```
b'<?xml version="1.0" encoding="UTF-8"?>\n<mets xmlns:mets="http://www.loc.gov/METS/" 
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xlink="http://www.w3.org/1999/xlink" 
xmlns:dc="http://purl.org/dc/elements/1.1/" TYPE="archival information package" OBJID="ark:/67531/12345">\n  
<metsHdr/>\n</mets>\n'
```

File: README.md [hunk 3]
```diff
@@ -3,6 +3,62 @@ pymets [![Build Status](https://travis-ci.org/unt-libraries/pymets.svg?branch=ma
...
 Python module for reading and writing METS files.
 
+```python
+>>> import io
+>>> from pymets import mets_structure, metsdoc
+>>> # To create a METS object
+>>> attributes = {
+            'TYPE': 'archival information package',
+            'OBJID': 'ark:/67531/12345',
+        }
```
Comment (ldko) [inline README.md]: Minor, but we don't need this much indentation here.

File: README.md [hunk 4]
```diff
@@ -3,6 +3,62 @@ pymets [![Build Status](https://travis-ci.org/unt-libraries/pymets.svg?branch=ma
...
+>>> m = mets_structure.Mets()
+>>> m.set_atts(attributes)
```
Comment (ldko) [inline README.md]: These two lines could be combined into the following:
```
>>> m = mets_structure.Mets(attributes=attributes)
```

File: README.md [hunk 5]
```diff
@@ -3,6 +3,62 @@ pymets [![Build Status](https://travis-ci.org/unt-libraries/pymets.svg?branch=ma
...
+>>> m.add_child(mets_structure.MetsHdr())
+>>> print(m.create_xml_string())
+>>> <?xml version="1.0" encoding="UTF-8"?>
+<mets xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:mets="http://www.loc.gov/METS/" 
+xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+TYPE="archival information package" OBJID="ark:/67531/12345">
+  <metsHdr/>
+</mets>
+
+>>> # To convert mets xml to python object
```
Comment (ldko) [inline README.md]: I would capitalize "METS XML" and "Python"

**Comment by ldko** [comment]
@madhulika95b any time you see an acknowledgments section on one of our projects that you work on, you should add your name!

**Comment by ldko** [comment]
Do you think we should also run tests with Python 3.6 via Travis?

**Comment by madhuBayy** [comment]
I can do that.

**Review by ldko — APPROVED** [review]
Looks good to me

**Review by somexpert — APPROVED** [review]
Everything looks good and the examples work :thumbsup:

**Pull request merged**